src/encoded/static/components/gene.js

```diff
@@ -119,6 +119,19 @@ class Gene extends React.Component {
                           </div>
                         : null}
 
+                        {context.locations && context.locations.length > 0 ?
+                            <div data-test="locations">
+                                <dt>Gene locations</dt>
+                                <dd>
+                                    <ul>
+                                        {context.locations.map((location, l) => (
```

Comment:
I tend not to use 'l' as an index because it can be misinterpreted as '1'. But the code is fine as-is.